CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to `c8`
+
+The `c8` project welcomes all contributions from anyone willing to work in good faith with other contributors and the community. No contribution is too small and all contributions are valued.
+
+## Issues
+
+- You can open [issues here](https://github.com/bcoe/c8/issues), please follow the template guide.
+- You can come over to the `node-tooling/c8` channel for discussions, [follow this link](https://devtoolscommunity.herokuapp.com/) to request for an invite.
+
+## Pull Requests
+
+Pull Requests are the way concrete changes are made to the code, documentation, dependencies, and tools contained in the `c8` repository.
+
+### Setting up your local environment
+
+- Make sure you have installed the latest version of Node.js
+- Fork this project on Github and clone your fork locally:
+  ```
+  $ git clone git@github.com:username/c8.git
+  $ cd c8
+  $ git remote add upstream https://github.com/bcoe/c8.git
+  $ git fetch upstream
+  ```
+- Create local branches to work within. These should also be created directly off of the master branch:
+  ```
+  $ git checkout -b my-branch -t upstream/master
+  ```
+- Make your changes
+- Run tests to make sure all is okay (everything should pass except snapshot):
+  ```
+  $ npm test
+  ```
+- Now update the snapshot
+  ```
+  $ npm run test:snap
+  ```
+- If all is passing, commit your changes following the [conventional commits guideline](https://www.conventionalcommits.org/)
```

Comment:
I'd argue this isn't strictly necessary since the maintainers can do this with the squash and merge function. IME this is a barrier to entry for beginners, so I'd put it in the Maintainers' guide.

Reply:
Makes sense, will move it.